DESCRIPTION

```diff
@@ -35,3 +41,10 @@ Suggests:
     BSgenome,
     BiocManager,
     BSgenome.Hsapiens.UCSC.hg38
+URL: https://github.com/GoekeLab/proActiv
```

Comment:
Addition of URL, biocViews and maintainer. 
I'm not sure what other relevant biocViews should be added, there isn't really anything to do with promoters, or alternative TSS. AlternativeSplicing or DifferentialSplicing?
https://bioconductor.org/packages/devel/BiocViews.html#___Software

Reply:
suggestions: Transcription, Gene Expression, Alternative Splicing, GeneRegulation, DifferentialSplicing, FunctionalGenomics,Epigenetics, Transcriptomics, RNASeq, Preprocessing